fs/copy.go

```diff
@@ -103,7 +103,9 @@ func copyDirectory(dst, src string, inodes map[uint64]string) error {
 	return nil
 }
 
-func copyFile(source, target string) error {
+// CopyFile copies the source file to the target.
+// The most efficient means of copying is used for the platform.
+func CopyFile(source, target string) error {
```

Comment:
We usually use LHS notation for the args in this package.

Should be `CopyFile(target, source string)`

Reply:
Fixed